websocket_push_session.go

```diff
@@ -27,13 +27,21 @@ import (
 	"go.uber.org/zap"
 )
 
+const (
+	// supportsErrorEventsQueryParam contains the name of the query parameter that can be passed in by the client to declare that
+	// it can handle error events
+	supportsErrorEventsQueryParam = "supportsErrors"
+)
```

Comment:
👀 👀 👀

Reply:
I just noticed the name is weird. it should be `enableErrors`

Reply:
Will integrate this in. This will impact the [`manipulate` PR](https://github.com/aporeto-inc/manipulate/pull/113) as well.

Reply:
Done ✅